horace_core/sqw/coord_transform/@aProjection/aProjection.m

```diff
@@ -67,11 +69,15 @@
         % true, though testing or the projection used to identify position
         % of q-dE point in q-dE space may set this property to false.
         do_3D_transformation;
+        % direct access to different parts of 4-component label celarray.
+        % sets up appropriate element of such array. Do not have a getter.
+        % Do retrievel label as a whole.
```

Comment:
Why no getter if this is needed? Fixed grammar, but have no idea what this is trying to tell me.

```suggestion
        % direct access to different parts of 4-component label cellarray.
        % sets appropriate element of this array. Does not have a getter.
        % Do retrieval label as a whole.
```